enterprise/dev/deployment-notifier/manifest_differ.go

```diff
@@ -57,6 +57,10 @@ func (m *manifestDeploymentDiffer) parseManifests() error {
 			// If the file is at the root, skip it. Applications are always in subfolders.
 			continue
 		}
+		// Skip READMEs and unrelated files
+		if len(elems) < 2 {
```

Comment:
```suggestion
		// Skip files in root directory - these are never relevant
		if len(elems) == 1 {
```

Should we also just filter out anything that's not `yaml|yml`?

also `.buildkite/pipeline.yaml` slips past this filter

Reply:
You're dead right, that's much better.